BUILD.bazel

```diff
@@ -476,6 +500,18 @@ genrule(
         mkdir -p python/ray/core/src/plasma &&
         cp $(location @plasma//:plasma_store_server) python/ray/core/src/plasma/ &&
         cp $(location //:raylet) python/ray/core/src/ray/raylet/ &&
-        mv python $(location ray_pkg)
-    """,
+        mv python $(location ray_pkg) &&
+        mkdir -p $(location ray_pkg)/src/ray/raylet &&
+        for f in $(locations //:raylet_library_java); do if [[ $$f == *".so"* ]]; then cp $$f $(location ray_pkg)/src/ray/raylet/"""
+        + select({
+            "@bazel_tools//src/conditions:windows": "libraylet_library_java.dll",
+            "@bazel_tools//src/conditions:darwin": "libraylet_library_java.dylib",
+            "//conditions:default": "libraylet_library_java.so"})
+        + "; fi done &&" +
```

Comment:
It seems that the default output file is always suffixed by `.so`, and you want to rename it with different suffixes based on the OS? 
It's weird that why it doesn't generate different suffixes in the first place.